server/handlers/component_handler.go

```diff
@@ -1269,6 +1272,75 @@ func (h *Handler) RegisterMeshmodelComponents(rw http.ResponseWriter, r *http.Re
 	go h.config.MeshModelSummaryChannel.Publish()
 }
 
+// swagger:route POST /api/meshmodel/nonRegisterEntity
+// Handle POST request for registering meshmodel components.
+//
+// Validate the given value with the given schema
+// responses:
+// 	200:MeshModelHostsWithEntitySummary
+
+// request body should be json
+// request body should be of Host format
+
+func (handler *Handler) NonRegisterEntity(responseWriter http.ResponseWriter, request *http.Request, _ *models.Preference, user *models.User, provider models.Provider) {
+	responseWriter.Header().Set("Content-Type", "application/json")
+	defer request.Body.Close()
+	registry.Mutex.Lock()
```

Comment:
reason for not closing the mutex because we want to display the log only once at the first load if we unlock it the log will be displayed everytime we switch to dashboard tab